modules/virt-configuring-openshift-root-CA-for-providers.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-openshift-root-CA-for-providers_{context}"]
```

Comment:
sorry for the annoying nit, but context statements and file names are supposed to be all lowercase, IIRC